packages/mgt-element/src/mock/MockGraph.ts

```diff
@@ -74,13 +75,21 @@ class MockMiddleware implements Middleware {
    */
   private _nextMiddleware: Middleware;
 
-  private static _baseUrl;
+  private static _baseUrl: string;
+
+  private static _session: SessionCache;
+
+  constructor() {
+    if (storageAvailable('sessionStorage')) {
+      MockMiddleware._session = new SessionCache();
```

Comment:
Shouldn't this be assigning to this?
```suggestion
      this._session = new SessionCache();
```

Reply:
Yes it should. The access modifiers used in the `MockMiddleware` class were complaining:

```javascript

Property '_session' does not exist on type 'MockMiddleware'. Did you mean to access the static member 'MockMiddleware._session' instead?
```
Using `private static propertyName` is causing errors in `public execute()` methods. Doing it like that works but it's just not elegant. I have updated the properties and names with no functional change.